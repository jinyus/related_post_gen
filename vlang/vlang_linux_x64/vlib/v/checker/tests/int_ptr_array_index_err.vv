pub fn read_int(bytes []u8, mut offset &int) int {
	defer {
		offset += 4
	}
	return bytes[offset]
}
