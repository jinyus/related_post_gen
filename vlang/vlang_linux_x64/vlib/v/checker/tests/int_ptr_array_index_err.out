vlib/v/checker/tests/int_ptr_array_index_err.vv:5:14: error: non-integer index `&int` (array type `[]u8`)
    3 |         offset += 4
    4 |     }
    5 |     return bytes[offset]
      |                 ~~~~~~~~
    6 | }
